tm2/pkg/crypto/keys/client/maketx.go

```diff
@@ -219,6 +221,7 @@ func ExecSignAndBroadcast(
 	io.Println("GAS USED:  ", bres.DeliverTx.GasUsed)
 	io.Println("HEIGHT:    ", bres.Height)
 	io.Println("EVENTS:    ", string(bres.DeliverTx.EncodeEvents()))
+	io.Println("TX HASH:   ", url.QueryEscape(base64.StdEncoding.EncodeToString(bres.Hash)))
```

Comment:
@zivkovicmilos During testing, I observed that when the hash returned in base64 format contains special characters like this one, `8y9J740MXZUhlgUv+Kxt+yVRqtTuFjHpNdLZPkZVhR4=`, the hash parsed from the HTTP server side converts special characters into spaces. This function will help transform special characters into their corresponding base64 string `8y9J740MXZUhlgUv%2BKxt%2ByVRqtTuFjHpNdLZPkZVhR4%3D` while preserving the integrity of the original string :